pkg/operator/controller/controller_test.go

```diff
@@ -25,6 +25,9 @@ import (
 	"strconv"
 	"strings"
 
+	"kubevirt.io/containerized-data-importer/pkg/monitoring/rules"
+	"kubevirt.io/containerized-data-importer/pkg/monitoring/rules/alerts"
```

Comment:
should be grouped with the other `kubevirt.io/containerized-data-importer` imports